api/src/main/java/com/graphhopper/util/Helper.java

```diff
@@ -458,4 +458,15 @@ public static final String underScoreToCamelCase(String key) {
 
         return sb.toString();
     }
+
+    /**
+     * Equivalent to java 8 String#join
+     */
+    public static String join(String delimiter, List<String> strings) {
+        StringJoiner joiner = new StringJoiner(delimiter);
```

Comment:
That would cause problem on Android compatibility, since [StringJoiner](https://developer.android.com/reference/java/util/StringJoiner) being a Java 8 class, was added in API level 24 (Android 7 Nougat).

Reply:
Oh right! Thanks for pointing this out, will replace.